mono/metadata/icall-def.h

```diff
@@ -124,6 +124,11 @@ HANDLES(NATIVEMETHODS_9, "SetProcessWorkingSetSize", ves_icall_Microsoft_Win32_N
 HANDLES(NATIVEMETHODS_10, "TerminateProcess", ves_icall_Microsoft_Win32_NativeMethods_TerminateProcess, MonoBoolean, 2, (gpointer, gint32))
 HANDLES(NATIVEMETHODS_11, "WaitForInputIdle", ves_icall_Microsoft_Win32_NativeMethods_WaitForInputIdle, gint32, 2, (gpointer, gint32))
 
+#if ENABLE_NETCORE // && UNIX
```

Comment:
the comment seems unrelated

Reply:
On non-Unix builds it P/Invokes directly to the Win32 API, so the comment makes sense.